opennms-doc/guide-admin/src/asciidoc/text/webui/heatmap/introduction.adoc

```diff
@@ -21,7 +21,8 @@ image:webui/heatmap/heatmap.png[]
                                                            default.
                                                                                                                         | `alarms`
 | `org.opennms.heatmap.defaultHeatmap`       | _String_  | This option defines which _Heatmap_ is displayed by default.
-                                                           The two valid options are `categories` or `foreignSources`   | `categories`
+                                                           The three valid options are `categories`, `foreignSources` and
+                                                           'services'.                                                  | `categories`
```

Comment:
Maybe just write `Valid options are: ...` So you do not have to change the numbers everytime the list changes.